File: core/repomap/testdata/sample.py
import os

def greet(name):
    return f"Hello {name}"

class Greeter:
    def __init__(self, prefix):
        self.prefix = prefix
    def greet(self, name):
        return self.prefix + " " + name

g = Greeter("Hi")
print(greet("world"))

File: core/repomap/testdata/sample.go
package sample

type Greeter interface {
	Greet(name string) string
}

type Hello struct {
	Prefix string
}

func (h *Hello) Greet(name string) string {
	return h.Prefix + " " + name
}

func NewHello(prefix string) *Hello {
	return &Hello{Prefix: prefix}
}

const Default = "Hi"
var Counter = 0
